examples/sap_hana_ha_simple_example/variables.tf

```diff
@@ -123,12 +123,19 @@ variable "network_tags" {
   default     = []
 }
 
+variable "public_ip" {
+  description = "Determines whether a public IP address is added to your VM instance."
+  default     = 1
```

Comment:
We should use `true` or `false` as values (and a `type = bool`).

Reply:
Thanks for the feedback. I'm happy to change this, I just had a look at the NetWeaver module before and that one is using an int and I was trying to make it the same behaviour. What do you think. Should we change it to bool here and accept the difference between the NW and HANA modules?

Reply:
Yeah I believe those modules were developed prior to Terraform v0.12's richer type support. Please go ahead and change it here - we can accept the difference between the modules.

Reply:
+1 makes sense. I will adapt